traitcuration/static/js/time-helper.js

```diff
@@ -0,0 +1,19 @@
+// This module contains utilities for date formatting and output.
+
+const timeFormat = "MMMM Do YYYY, H:MM ";
```

Comment:
I can see what this looks like in the final deployment, but it would be helpful to also include a comment with an example of the final formatted timestamp here in the code